src/views/ApartmentApp/components/StudentApplication/index.js

```diff
@@ -224,7 +216,7 @@ const StudentApplication = ({ userProfile, authentication }) => {
   const addApplicant = async (newApplicantUsername) => {
     try {
       // Get the profile of the selected user
-      const newApplicantProfile = await user.getProfileInfo(newApplicantUsername);
+      let newApplicantProfile = await user.getProfileInfo(newApplicantUsername);
```

Comment:
Is there a reason this was changed from `const` to `let`? It doesn't seem like you mutate it later on, so presumably `const` is the correct declaration.

Reply:
```suggestion
      const newApplicantProfile = await user.getProfileInfo(newApplicantUsername);
```